### PR TITLE
Script to automate RC branch creation

### LINKED ISSUE #90: Create script to help in the release process
It would be really nice to have a script that helps in the release process:

 0. Work only on `dev` branch
 1. Check for next release (or ask the user for the milestone)
 2. Get information about the milestone:
    + if there open PR, complain, we are not going to release if there is pending work
    + for every closed PR, get the `changelog` information
 3. Write changelog information into the `CHANGELOG` file (it should be a new entry)
 4. Modify version string (check it is greater than existing one)
 5. Commit these changes into `dev`
 6. Open release candidate branch `rc-<version>`
 7. Create PR to `master`

### PR DESCRIPTION
**Pull request by jgsogo**
First approach, probably we should push the changes to `dev`, push the `rc` branch and create the PR to `master`.

closes #90 
on top of #119

### CODE REVIEW

File: make_rc.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
+
```
Comment (SSE4) [inline make_rc.py]: shebang

File: make_rc.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+            os.system("git add Conan.VisualStudio/source.extension.vsixmanifest")
+
+            if query_yes_no("Commit change to 'dev' branch"):
+                os.system('git commit -m "close milestone {}"'.format(next_release))
+                os.system('git push')
+
+                os.system('git checkout -b rc-{}'.format(next_release))
+                os.system('git push')
+
+                sys.stdout.write("Now commit this branch and make the PR to master")
```
Comment (SSE4) [inline make_rc.py]: it's already committed and pushed, what do you mean?

File: make_rc.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+import re
+
+from functools import partial
+from datetime import date
+
+try:
+    from github import Github
+except ImportError:
+    sys.stderr.write("Install 'pip install PyGithub'")
+    exit()
```
Comment (SSE4) [inline make_rc.py]: it needs to use sys.exit(1) for failures to indicate an error (here and below)

File: make_rc.py (hunk 2 above)
Comment (jgsogo) [reply make_rc.py]: All of this should happen in the repo, not in the local working copy

**Comment by jgsogo** [comment]
Let's merge this in the next 1.2.1 version, dedicated to _CI_ and related stuff

**Pull request merged**